community/resources/resources.md

```diff
@@ -37,14 +37,19 @@ We have channels for each project. These channels are recommended for new member
 - [github][]: Support for GitHub issues, pull requests, and processes
 - [public-backlog][]: Discussions of the Magento 2 backlog
 - [devdocs][]: Documentation contribution support
+- [cloud][]: Open chat for cloud questions
+- [graphql][]: Open chat for graphql contribution
+- [pwa][]: Open chat for Progressive Web Applications contribution
+- [msi][]: Open chat for Multi-Source Inventory contribution
+- [mftf][]: Open chat for Magento Functional Testing Framework(MFTF) contribution
 
 ## Resources and guides
 
 - *Video* [How to contribute during Magento Contribution Days][57] by [Max Pronko][35]
 - *GitHub* [Magento Resources][23] curated list by [Alessandro Ronchi][30]
 - *Blog posts* [Thoughts on Magento, PHP, JavaScript, Laravel, React, Docker, and user interface design][56] by [Mark Shust][38]
-- *Videos* [Mage2.tv][24] 
-- *Podcast* [MageTalk][25] by [Phillip Jackson and Kalen][27] 
+- *Videos* [Mage2.tv][24]
```

Comment:
I should add `by Vinai Kopp`

Reply:
@osrecio I added vinai kopp name for Mage2.tv